node/pkg/db/governor_db.go

```diff
@@ -0,0 +1,199 @@
+package db
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"time"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+	"github.com/certusone/wormhole/node/pkg/vaa"
+	"github.com/dgraph-io/badger/v3"
+
+	"go.uber.org/zap"
+)
+
+type Transfer struct {
+	Timestamp      time.Time
+	Value          uint64
+	OriginChain    vaa.ChainID
+	OriginAddress  vaa.Address
+	EmitterChain vaa.ChainID
+	EmitterAddress vaa.Address
+	MsgID          string
+}
+
```

Comment:
Could we have unit-tests in governor_db_test.go for the functions defined in governor_db.go?